demo/dynamic_embedding/movielens-1m-keras-with-horovod/movielens-1m-keras-with-horovod.py

```diff
@@ -184,6 +184,25 @@ def embedding_out_split(embedding_out_concat, input_split_dims):
   return embedding_out
 
 
+class Bucketize(tf.keras.layers.Layer):
```

Comment:
What is it used for?

Reply:
In here https://github.com/tensorflow/recommenders-addons/blob/743cc4b10993aec388c7bcb0174e6bff89b90f6a/demo/dynamic_embedding/movielens-1m-keras-with-horovod/movielens-1m-keras-with-horovod.py#L375